Gujarati/git.txt

```diff
@@ -0,0 +1,2 @@
+I have also taken part
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.